core/domain/exp_services.py

```diff
@@ -927,11 +927,10 @@ def update_exploration(
 
     if (is_suggestion and (
             not commit_message or
-            not commit_message.startswith(
-                feconf.COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX))):
+            not commit_message.startswith('Accepted '))):
```

Comment:
Not sure why these checks where added in the first place, should we remove it? Or should I add a regex check?

Reply:
Fixed!